src/main/com/example3/ArrayRNG.java

```diff
@@ -0,0 +1,37 @@
+package main.com.example3;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class ArrayRNG {
+
+    public static void main(String[] args) {
+
+        int array[] = new int[49];
+        int array2[] = new int[6];
+        array = fillArray(false,array);
```

Comment:
metoda fillArray by mela neco naplnit ... podle takoveho nazvu neocekavam, ze mi bude neco vracet, s cim chci dale pracovat

Reply:
Přejmenováno na getFilledArray, už je to přesnější, co to bude dělat